### PR TITLE
BUILD-3774: remove checked-in libs and add logic to download libs from our storages

### PR DESCRIPTION
**Pull request by ManuelJNunez**
In this PR I'm removing the checked-in libraries to improve the maintanibility of this repository. Instead of that, I added the `config.yaml` file the lists that should be downloaded and the `download_latest_libs.py` to read the file and pull them from the storage.

### CODE REVIEW

File: config.yaml
Comment (ManuelJNunez) [inline config.yaml:1]: Please review carefully this file, let me know if the version is correct.

File: resources/scripts/requirements.txt
Comment (ManuelJNunez) [inline resources/scripts/requirements.txt:1]: These are the requirements to run `download_latest_libs.py`

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,6 @@ docs/_build/
 # node.js dependencies 
 node_modules/
 
+# Libraries directory
+lib/
```
Comment (samuelraeburn) [inline .gitignore:57]: Why add to the gitignore? We have deleted it, right?

Comment (ManuelJNunez) [reply .gitignore:57]: Just in case you accidentally commit them.

**Review by samuelraeburn — APPROVED** [review]

**Review by lulivi — APPROVED** [review]

**Pull request merged**